File: tools/dbench/tasks/m1_oldest.py
best_name, best_age = "", -1
with open("users.txt") as f:
    for line in f:
        parts = line.split()
        if len(parts) != 3:
            continue
        try:
            age = int(parts[2])
        except ValueError:
            continue
        if age > best_age:
            best_name, best_age = parts[1], age
print(f"{best_name} {best_age}")

File: tools/dbench/answers/rust_haiku/s3/m1_oldest.rs
use std::fs;

fn main() {
    let txt = fs::read_to_string("users.txt").unwrap_or_default();
    let mut best_age = -1;
    let mut best_name = "";

    for line in txt.lines() {
        let parts: Vec<&str> = line.split_whitespace().collect();
        if parts.len() == 3 {
            if let Ok(age) = parts[2].parse::<i32>() {
                if age > best_age {
                    best_age = age;
                    best_name = parts[1];
                }
            }
        }
    }

    println!("{} {}", best_name, best_age);
}
